mm2src/coins/utxo/utxo_block_header_storage/mod.rs

```diff
@@ -323,7 +322,7 @@ mod wasm_test {
 
     wasm_bindgen_test_configure!(run_in_browser);
 
-    const FOR_COIN: &str = "RICK";
+    const FOR_COIN: &str = "tBTC";
```

Comment:
Is there a reason why you switch from using `RICK` here ?

Reply:
Yes, since https://github.com/KomodoPlatform/atomicDEX-API/pull/1647 RICK and MORTY block headers are deserialized another way, so I got failed tests.

Reply:
ok cool. thank you